src/safeposix/syscalls/fs_calls.rs

```diff
@@ -104,6 +104,7 @@ use crate::safeposix::shm::*;
 impl Cage {
     /// ## ------------------OPEN SYSCALL------------------
     /// ### Description
+    ///
```

Comment:
- Refactored `open_syscall` to make it in sync with the implementation of other sys_calls.
- Added `Panics` section for it.